test/integration/forms/action.js

```diff
@@ -1988,4 +1988,375 @@ context('Patient Action Form', function() {
       .contains('Submit')
       .should('not.exist');
   });
+
+  specify('user has only work:owned:submit permission', function() {
```

Comment:
Quick test overview:

1. Set the current clinician to have the `work:owned:submit` permission.
2. Load a patient dashboard with two actions. One that is owned by the current clinician and another that is not.
3. Then I click through to both of those action's forms and verify that the submit button is properly restricted. And that the form was rendered as read only.

Reply:
May potentially name these tests something different too. Not sure yet though...

Reply:
This has been updated to not include `"only"`.